app/models/git/exercise.rb

```diff
@@ -33,6 +33,17 @@ def synced_git_sha
       commit.oid
     end
 
+    def valid_submission_filepath?(filepath)
+      return false if filepath.match?(%r{[^a-zA-Z0-9_./-]})
+      return false if filepath.starts_with?(".meta")
```

Comment:
Do we want to exclude `.docs` files too?

Reply:
Sure :)